ci-operator/templates/cluster-launch-installer-e2e.yaml

```diff
@@ -164,17 +183,30 @@ objects:
       volumeMounts:
       - name: shared-tmp
         mountPath: /tmp
+      # TODO
+      # This is how I created cluster-profile from local system and from openshift/release dir
+      # oc create secret generic dev-cluster-profile --from-file=cluster/test-deploy/aws/openshift.yaml -n ns-name
       - name: cluster-profile
         mountPath: /etc/openshift-installer
       - name: artifacts
         mountPath: /tmp/artifacts
+      # TODO
+      # This is how I uploaded creds from local system:
+      # oc create secret -n ns-name generic cluster-secrets-aws --from-file=/path/to/secret/pull-secret --from-file=/path/to/secret/license --from-file=/path/to/secret/credentials -o yaml --dry-run | oc apply -n ns-name -f -
+      - name: license-pull-secret-aws
+        mountPath: /tmp/license-pull-secret-aws
       env:
       - name: INSTANCE_PREFIX
         value: ${NAMESPACE}-${JOB_NAME_HASH}
+      - name: BAZEL_IMAGE
+        value: "quay.io/coreos/tectonic-builder:bazel-v0.3"
       - name: TYPE
         value: ${CLUSTER_TYPE}
+      # TODO
+      # This is how I ran from local system
+      # Uploaded from local system, via secret cluster-secrets-aws
       - name: AWS_SHARED_CREDENTIALS_FILE
-        value: /etc/openshift-installer/.awscred
+        value: /tmp/cluster/credentials
```

Comment:
This will break current setup!! I will remove this but have it here to enable local testing of this PR..This is how I got my aws credentials file to succeed from my local system.. I used the file ``~/.aws/credentials`  NOT `~/.awscred`

Reply:
```cat ~/.awscred
AWSAccessKeyId=AKIXXXX
AWSSecretKey=XXXXXX
AWSKeyPairName=libra
AWSPrivateKeyPath=~/.ssh/libra.pem
```
and
```
cat ~/.aws/credentials
[default]
aws_access_key_id = AKIXXXXXXX
aws_secret_access_key = XXXXXXX
```